satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -377,6 +377,102 @@ read limitoffset (
     orderby desc bucket_usage.rollup_end_time
 )
 
+//--- used serials ---//
+
+model serial_number (
+	key    id
+	unique serial_number
+
+	field id            serial
```

Comment:
is this `id` the storage_node_id?

Reply:
this is the serial number id, not sure how to name it better

Reply:
ohh that makes sense. do we also need the storage_node_id here?

Reply:
that's in the `used_serials` table

Reply:
reason for not using `serial_number` itself as the key in `used_serials` is because the serial is 4bytes, but serial number itself is 16bytes. given 100 used entries per limit, we end up saving 1.2kb

Reply:
ok I think I understand.  The `serial_number` gets created when orderlimits are created for uplink so we store the bucket_id in that table.  And the `used_serial` gets created when the SN sends orders, so we store the store_node_id with that. 

Also that answers my question why store an additional `id` instead of `serial_num` itself, the 4bytes vs 16 bytes. thanks.